docs/conf.py

```diff
@@ -67,7 +67,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+# language = None
```

Comment:
I've forgotten what the deal is with this.  Is this change fixing an unrelated docs bug?

Reply:
Modern Sphinx complains about this. Don't ask me why.

Reply:
OK, I'll consider that an unrelated docs bug fix then!